cita-executor/core/src/executive.rs

```diff
@@ -326,11 +326,16 @@ impl<'a, B: 'a + StateBackend> Executive<'a, B> {
 
         let schedule = Schedule::new_v1();
 
+        let gas_used_by_data_length = match t.version {
+            0...2 => 0,
+            _ => t.data.len() * 4,
+        };
+
         let base_gas_required = match t.action {
             Action::Create => schedule.tx_create_gas,
             Action::GoCreate => schedule.tx_create_gas,
             _ => schedule.tx_gas,
-        };
+        } + gas_used_by_data_length;
```

Comment:
I think it's better to add a new variable named like `data_gas_required = base_gas_required  + gas_used_by_data_length`, but not channging the original `base_gas_required` 's definition